docs/deployment-environment-variables.md

```diff
@@ -3,26 +3,33 @@ id: deployment-environment-variables
 title: Environment Variables
 ---
 
-Various services require environment variables to operate.
+Various services require environment variables to operate. While they may appear overwheling, this setup guide aims to alleviate some of the concern. It is recommended you start with the required variables and work through this guide so all are properly setup.
```

Comment:
```suggestion
Various services require environment variables to operate. While they may appear overwhelming, this setup guide aims to alleviate some of the concern. 
```
I feel like the required/optional parts below are well laid out enough that we don't need to emphasize that